app/scripts/modules/core/src/pipeline/manualExecution/Parameters.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { FormikProps } from 'formik';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+
+import { HelpField } from 'core/help';
+import { IParameter, IPipelineCommand } from 'core/domain';
+import { FormikFormField, ReactSelectInput, TextInput } from 'core/presentation';
+
+import 'react-day-picker/lib/style.css';
+
+export interface IParametersProps {
+  formik: FormikProps<IPipelineCommand>;
+  parameters: IParameter[];
+}
+
+export class Parameters extends React.Component<IParametersProps> {
+  constructor(props: IParametersProps) {
+    super(props);
+  }
+
+  private dateSelected = (date: Date, name: string): void => {
+    this.props.formik.setFieldValue('parameter.' + name, date.toISOString().slice(0, 10));
+  };
+
+  private shouldInclude = (p: IParameter) => {
+    const { values } = this.props.formik;
+    if (p.conditional) {
+      const comparingTo = values.parameters[p.conditional.parameter];
+      const value = p.conditional.comparatorValue;
+      switch (p.conditional.comparator) {
+        case '>':
+          return parseFloat(comparingTo) > parseFloat(value);
+        case '>=':
+          return parseFloat(comparingTo) >= parseFloat(value);
+        case '<':
+          return parseFloat(comparingTo) < parseFloat(value);
+        case '<=':
+          return parseFloat(comparingTo) <= parseFloat(value);
+        case '!=':
+          return comparingTo !== value;
+        case '=':
+          return comparingTo === value;
+      }
+    }
+    return true;
+  };
+
+  public render() {
+    const { parameters } = this.props;
+    const hasRequiredParameters = parameters.some(p => p.required);
+    const hiddenParameters = new Set<string>();
+    parameters.forEach(p => {
+      if (p.conditional) {
+        const include = this.shouldInclude(p);
+        if (!include) {
+          hiddenParameters.add(p.name);
+        }
+      }
+    });
+    return (
+      <>
+        <p className="manual-execution-parameters-description">
+          This pipeline is parameterized. Please enter values for the parameters below.
+        </p>
+        {parameters &&
+          parameters.map((parameter, i) => {
+            if (hiddenParameters.has(parameter.name)) {
+              return <> </>;
+            } else {
+              return (
+                <div className="form-group" key={i}>
+                  <div className="col-md-4 sm-label-right break-word">
+                    {parameter.name}
+                    {parameter.required && <span>*</span>}
+                    {parameter.description && <HelpField content={parameter.description} />}
+                  </div>
+                  {!parameter.hasOptions && parameter.constraint === 'date' && (
```

Comment:
This might be an embarrassing question but how do you add a constraint to a parameter? I just manually edited the pipeline JSON to test the new date picker (which looks great!)

Reply:
`constraint` is something that is specific to implementation in Netflix. I asked before and was told I should preserve the functionality